frontend/lib/resolution/disambiguation.cpp

```diff
@@ -64,10 +76,30 @@ struct DisambiguationContext {
       callInScope(callInScope), callInPoiScope(callInPoiScope),
       explain(explain)
   {
+    isMethodCall = call->isMethodCall();
+
+    // this is a workaround -- a better solution would be preferred
+    if (parsing::idIsInInternalModule(context, callInScope->id())) {
+      useOldVisibility = true;
+    }
+
+    // this is a workaround -- a better solution would be preferred.
+    // this function seems to be created in a way that has problems with
+    // the visibility logic in disambiguation.
+    if (call->name() == UniqueString::get(context, "_getIterator")) {
```

Comment:
You can just write `if (call->name() == "_getIterator"))`

Reply:
will adjust in a later, trivial PR

Reply:
fixed in https://github.com/chapel-lang/chapel/pull/23838